<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CmdSend</class>
 <widget class="QWidget" name="CmdSend">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>156</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="statusGroupBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>0</y>
     <width>150</width>
     <height>155</height>
    </rect>
   </property>
   <property name="title">
    <string>Arm Status</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>26</y>
      <width>134</width>
      <height>124</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="btnHome">
         <property name="text">
          <string>Home</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnEnable">
         <property name="text">
          <string>Enable</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnEmergencyStop">
         <property name="text">
          <string>Emergency Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btn2DPlane">
         <property name="maximumSize">
          <size>
           <width>62</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>2DPlane</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn3DSpace">
         <property name="maximumSize">
          <size>
           <width>62</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>3DSpace</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>0</y>
     <width>150</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Net Settings</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>131</width>
      <height>96</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="labelIP">
           <property name="text">
            <string>IP</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="IPLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="labelPort">
           <property name="text">
            <string>Port</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="portSpinBox">
           <property name="maximum">
            <number>9999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="btnConnect">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="teachGroupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Teach</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>26</y>
      <width>103</width>
      <height>89</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btnRecord">
       <property name="text">
        <string>Record</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStopRecord">
       <property name="text">
        <string>StopRecord</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnReappear">
       <property name="text">
        <string>Reappear</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
